go.mod1

```diff
@@ -38,3 +38,8 @@ require (
 	google.golang.org/grpc v1.27.1
 	sourcegraph.com/sourcegraph/appdash v0.0.0-20190731080439-ebfcffb1b5c0
 )
+
+replace (
+	cloud.google.com/go/storage => github.com/3pointer/google-cloud-go/storage v1.6.1-0.20210108125931-b59bfa0720b2
+	golang.org/x/sys => golang.org/x/sys v0.0.0-20200625212154-ddb9806d33ae
```

Comment:
but why have this replace?

Reply:
It used to avoid some complie error on my mac os when I try to upgrade  cloud.google.com/go/storage from `v1.6.0` to `v1.12.0`. but for now it can be removed. because I have picked googleapis/google-cloud-go#3509 to v1.6.0 already.